package.json

```diff
@@ -7,8 +7,10 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "eslint ./src --ext .js,.ts",
+    "lint-fix": "eslint ./src --fix --ext .js,.ts",
     "gen": "ng-openapi-gen --input http://localhost:8080/atlas/v3/api-docs -o generated/api",
     "postgen": "prettier generated/api --write",
+    "gen-api": "ng-openapi-gen --input http://localhost:8080/atlas/v3/api-docs -o generated/api && prettier generated/api --write",
```

Comment:
This shouldn't be necessary gen + postgen does the same

Reply:
Well typing one command is faster than two. This is just to make sure both commandy have been run and I dont foget one.

Reply:
after running `gen`, `postgen` is executed automatically anyway as far as I know

Reply:
> after running `gen`, `postgen` is executed automatically anyway as far as I know

Looks like it. Did remove the command then.